config/locales/fr.yml

```diff
@@ -30,10 +33,16 @@ fr:
     report_abuse: Signaler un abus
     search: Recherche
     see_all: Voir tout
+    series: Séries
     trending_on: Populaire sur
     view: Vue
     week: Semain
     year: Anée
+  dashboard:
+    empty_follower: Vous n'avez pas encore d'abonnés...
+    empty_post: C'est ici que vous pouvez gérer vos messages, mais vous n'avez encore rien écrit
+    upload_a_video: Upload a video
```

Comment:
This should be `Charger une vidéo` or `Télécharger une vidéo`

Reply:
Sure, thank you. 👍